src/components/TagModal.vue

```diff
@@ -173,7 +173,10 @@ export default {
 		async createTag(event) {
 			this.editing = true
 			const displayName = event.target.querySelector('input[type=text]').value
-
+			if(this.$store.getters.getTags.some(tag=>tag.imapLabel === "$"+displayName)){
+				showError(this.t('mail', 'Duplicity error, unable to create the tag.'))
```

Comment:
```suggestion
				showError(this.t('mail', 'Tag already exists'))
```

A little bit more specificity for the users.